src/app/shell/shell-routing.module.ts

```diff
@@ -83,6 +84,12 @@ const routes: Routes = [
     canLoad: [CreateProviderGuard],
     canDeactivate: [CreateProviderGuard, CreateGuard]
   },
+  {
+    path: 'create-provider-admin/:param', component: CreateProviderAdminComponent,
+    loadChildren: () => import('./personal-cabinet/provider/provider.module').then(m => m.ProviderModule),
+    canLoad: [ProviderGuard],
```

Comment:
why? Maybe I am wrong, but what is the role for providerAdmin? is he admin or is he a provider? This guard works only for the provider role.